### PR TITLE
treehouses led onam (fixes #700)

### LINKED ISSUE #700: treehouses led <newholiday>

### PR DESCRIPTION
**Pull request by Neethu-Mohan**
Fixed issue #700. Added new festival to the file led.sh

### CODE REVIEW

File: modules/led.sh [hunk 1]
```diff
@@ -37,6 +37,17 @@ function led {
...
+  elif [ "$color" = "Onam" ]; then
+	checkroot
+	echo "leds are set to Onam mode."
+	echo "Look at your Rpi leds, both leds will be in this pattern..."
+	echo "green LED :5 blink"
+	echo "Both LED : 1 sec off"
+	echo "red LED :5 blink"
+	echo "green LED :5 blink"
+	echo "Both LED : 1 sec"
+	echo "red LED :5 blink"
```
Comment (LordJashin32) [inline modules/led.sh]: We use spaces, not tabs. Just 2 spaces for each level of indentation.

Comment (kjong) [reply modules/led.sh]: Also, a bit pedantic but could you make sure that your spaces and capitalization are consistent?

For example, you have `echo "green LED :5 blink"`, which could be changed to `echo "Green LED: 5 blink"`, and so on.

Also, `Onam` should probably be changed to `onam`.

**Comment by phamduchongan93** [comment]
@Neethu-Mohan some error at the end. 
```
pi@pinode1:~/cli $ sudo ./cli.sh led onam                                                                                                        
leds are set to onam mode.                                                                                                                       
Look at your Rpi leds, both leds will be in this pattern...
Green LED: 5 blink
Both LED: 1 sec off
Red LED: 5 blink
Green LED: 5 blink
Both LED: 1 sec
Red LED: 5 blink
Error: led 'onam' is not present 

```

**Comment by dogi** [comment]
> @Neethu-Mohan some error at the end.
> 
> ```
> pi@pinode1:~/cli $ sudo ./cli.sh led onam                                                                                                        
> leds are set to onam mode.                                                                                                                       
> Look at your Rpi leds, both leds will be in this pattern...
> Green LED: 5 blink
> Both LED: 1 sec off
> Red LED: 5 blink
> Green LED: 5 blink
> Both LED: 1 sec
> Red LED: 5 blink
> Error: led 'onam' is not present 
> ```

@phamduchongan93 nice catch ;)

https://github.com/treehouses/cli/issues/706
error not part of this PR

**Pull request merged**